src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/HttpCacheProxyNotModifiedResponse.java

```diff
@@ -118,10 +121,10 @@ private void onBegin(
         final HttpBeginExFW httpBeginFW = extension.get(factory.httpBeginExRO::wrap);
         final ArrayFW<HttpHeaderFW> responseHeaders = httpBeginFW.headers();
 
-        DefaultCacheEntry cacheEntry = factory.defaultCache.supply(requestHash);
+        DefaultCacheEntry cacheEntry = factory.defaultCache.supply(requestHash, requestURL);
```

Comment:
Can the second argument be changed to `requestHashWithoutQuery` instead of passing `requestURL`?